change/@ni-nimble-components-e882872b-5836-48a7-83b6-370d8c95d411.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We also talked about a few other tasks. Could you remind us what's the plan for these?
1. tooltip for basic text column headers
2. tooltip for rich content column headers (or, if it's a client responsibility, docs and tests to ensure it works)
3. documenting tooltip guidelines for column authors

Reply:
1. I [submitted a change](https://github.com/ni/nimble/pull/1093) to the `table-column-width-api` branch that uses the column header's `textContent` as its `title`. I forgot to write a test for this, but I will do that.

2. For icon-only headers, it has to be up to the client to provide tooltip text, and I think the expectation is that they set `title` themselves. There aren't currently docs for this, so I'm not sure where to make a note about this. As far as tests, I don't think there's anything to test for the icon-only case. For headers with icons and text together, I verified that the tooltip we set automatically does not interfere with any tooltip that they set for the icon.

3. Again, not sure where to document this.

Reply:
For icon headers, let's add a title to the storybook places that we use this pattern.

Can you make a note of 2 and 3 in upcoming work items #1044 and #976?

Reply:
The icon header in the table story already has a `title`. I've added a note to the two work items you referenced.